docs/design/seccomp.md

```diff
@@ -202,11 +202,11 @@ use annotations instead of extending the API with new fields.
 In the alpha version of this feature we will use annotations to store the
 names of seccomp profiles.  The keys will be:
 
-`security.alpha.kubernetes.io/seccomp/container/<container name>`
+`container.seccomp.security.alpha.kubernetes.io/<container name>`
 
 which will be used to set the seccomp profile of a container, and:
 
-`security.alpha.kubernetes.io/seccomp/pod`
+`seccomp.security.alpha.kubernetes.io/pod`
```

Comment:
Yeah, I wasn't suggesting /container-foo. I was hoping to be consistent by having both pod and container be part of the prefix, but validation won't allow for that.